wordpress-mobile/WordPress-Android
# Zendesk WIP: New Help Page

This PR updates the design of the Help page. Since it was basically a small re-write, I took the opportunity to convert the activity to Kotlin. Aside from porting the existing functionality to the new design, the only addition is "Contact email" which is implemented in #7825. That PR needs to be merged before this PR can be reviewed. (and the base branch needs to be updated)

To Test:

* Make sure the existing functionality works exactly the same way it used to (basically opening new pages 🤷‍♂️ )
* If you do a fresh install, you should see a "Not Set" for the "Contact email" but when you tap on it, the dialog will be pre-populated the same way it'll be when you tap on the "Contact us" button.
* Make sure saving the dialog updates the "Contact email" field

Here are some screenshots:

![screenshot_1527624135](https://user-images.githubusercontent.com/662023/40683085-6c73c304-635b-11e8-97a9-eae461ff177c.png)
![screenshot_1527624918](https://user-images.githubusercontent.com/662023/40683104-7de5c0c4-635b-11e8-9dc8-54d6f7776692.png)
![screenshot_1527624123](https://user-images.githubusercontent.com/662023/40683109-82e58ff0-635b-11e8-8f45-0c8799b3db09.png)

@SylvesterWilmott The Zeplin project had the page title `Support` whereas we used to have `Help and Support`. The title for the button of support page is `Help & Support`. Could we pick one of them and just use it in both the button title and the page title? I prefer the ones that include `Help` fwiw.
